qemu/tests/cfg/live_snapshot.cfg

```diff
@@ -70,3 +66,8 @@
                         stop_cmd = "killall -g stress"
                     before_start = "load_stress"
                     after_finished = "unload_stress reboot verify_alive"
+                - reboot:
```

Comment:
Just according to the test plan, reboot is one scenario under payload.
There are 4 scenarios under payload:
1. stress 2. reboot 3. stop_cont 4. installation